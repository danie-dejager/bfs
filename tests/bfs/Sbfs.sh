invoke_bfs -Sbfs -s basic >"$OUT"
diff_output
