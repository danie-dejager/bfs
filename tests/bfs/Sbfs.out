basic
basic/a
basic/b
basic/c
basic/e
basic/g
basic/i
basic/j
basic/k
basic/l
basic/c/d
basic/e/f
basic/g/h
basic/j/foo
basic/k/foo
basic/l/foo
basic/k/foo/bar
basic/l/foo/bar
basic/l/foo/bar/baz
